server/sched.go

```diff
@@ -616,3 +688,18 @@ func (s *Scheduler) unloadAllRunners() {
 		}
 	}
 }
+
+// If other runners are loaded, make sure the pending request will fit in system memory
+// If not, pick a runner to unload, else return nil and the request can be loaded
+func (s *Scheduler) maybeFindCPURunnerToUnload(req *LlmRequest, ggml *llm.GGML, gpus gpu.GpuInfoList) *runnerRef {
```

Comment:
Should we rename `GpuInfoList` to `ProcessorInfoList` or similar (can be in a follow up)? Given this is fully talking about system memory the `gpu` naming had me a bit confused

Reply:
Good call out.  Given I've got a bunch of other pending changes queued up on top of this, lets do this rename in a follow up to reduce merge rebase conflicts.

Reply:
Sounds great